k9mail/src/main/java/com/fsck/k9/activity/compose/RecipientMvpView.java

```diff
@@ -202,7 +202,13 @@ public void setCcVisibility(boolean visible) {
     }
 
     public void setBccVisibility(boolean visible) {
+        boolean bccAlreadyVisible = isBccVisible();
         bccWrapper.setVisibility(visible ? View.VISIBLE : View.GONE);
+        if(visible && !bccAlreadyVisible){
```

Comment:
This removes all recipients and re-adds them, fixing the symptom but not the cause. Do you know what causes the underlying problem?

Also, this is more logic than an MvpView class should have